arch/riscv/Kconfig.isa

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) 2022 Carlo Caione <ccaione@baylibre.com>
+# SPDX-License-Identifier: Apache-2.0
+
+menu "RISCV ISA and extensions"
+
+choice
+	prompt "Core ISA"
+	default RISCV_ISA_RV64I if 64BIT
+	default RISCV_ISA_RV32I
+
+config RISCV_ISA_RV32I
+	bool "RV32I Base Integer Instruction Set - 32bit"
```

Comment:
These should not have prompts since they are not meant to be user configurable (via menuconfig), and are intended to be only selected via the SoC Kconfigs, which by the way should not be user selectable either because they are only meant to be selected by the board Kconfig.

Reply:
ok